Host a Static Website with Hugo and NGINX.md

```diff
@@ -326,7 +325,7 @@ Press the **Escape** key, then press **:wq** to end editing of the config file.
 {{< /note >}}
 
 ### Vim Editor Basics
-
+{{Rohan: This section really doesn't belong in the scope of this article. Replace with an external link to a similar VI quick-start.}}
```

Comment:
Yeah, the article is already too long, so this section is out of scope. Provide an external link in the "Configuring Nginx" section.

Reply:
Interestingly, I agree. The issue is the Vim site doesn't even have a section with Vim Editor basics. I'll see if there are any good reputable options. I like the idea of a link. Issue is a reputable site. Should be Vim site but no such content.

Reply:
What would be nice is to (later possibly) point this to your own blog of VIM Basics.